zbuf/mux.go

```diff
@@ -0,0 +1,58 @@
+package zbuf
+
+import (
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/zio"
+)
+
+func Wrap(label string, batch Batch) Batch {
```

Comment:
There's already a function in this package called WrapBatch so this really wants another name. Maybe Label?

Reply:
Yeah that's confusing Label/Unlabel sounds good.